src/data/products.ts

```diff
@@ -0,0 +1,19 @@
+export const products = [
+  {
+    id: 123,
+    name: 'BBQ 양념치킨+크림치즈볼+콜라1.25L',
+    imageURL:
+      'https://st.kakaocdn.net/product/gift/product/20231030175450_53e90ee9708f45ffa45b3f7b4bc01c7c.jpg',
+    price: {
+      basicPrice: 29000,
+      discountRate: 0,
+      sellingPrice: 29000,
```

Comment:
큰 숫자는 가독성을 위해 numierc sperator를 사용해보셔도 좋을거 같아요
```suggestion
      sellingPrice: 29_000,
```